server.go

```diff
@@ -65,6 +65,10 @@ import (
 	stats "github.com/libp2p/go-libp2p-gorpc/stats"
 )
 
+const (
+	ContextKeyRequestSender = "request_sender"
```

Comment:
If public, please document the usage to have nice godoc. But having an accessor, I am not sure it is necessary that it is public.